src/arrayMethodFind.js

```diff
@@ -5,7 +5,12 @@
  */
 function applyCustomFind() {
   [].__proto__.find2 = function(callback) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this)) {
+        return this[i];
+      }
+    }
+    return undefined;
```

Comment:
new line before return,
it's not necessary to return undefined since it returns this by default